ec2.md

```diff
@@ -34,6 +36,16 @@ hypervisor: ec2
       nfs_server: none
       consoleport: 443
 
+## using role ###
+
+### No fog file needed ###
+### Update CONFIG section of  ec2 hosts file ###
+    CONFIG:
+      use_iam_role: true
+      nfs_server: none
```

Comment:
I think that you should leave off the `nfs_server` and `console_port` here, as they are not part of the pertinent config settings here.

Based on your comment, I think this should be renamed, perhaps something closer to what it actually changes; how does something like `anonymous_creation` or `unauthenticated_provisioning` sound? Naming things is hard...

Reply:
@tvpartytonight I can remove the `nfs_server: none` it was more to show exactly where this parameter should be placed based on the example above it.

As far as renaming, I'm not sure I'm following what you mean. If you are not using credentials and are using a iam_role then this parameter tells you it's going to ignore any credentials and try to use the iam_role associated with your current node to spin up ec2 instances. That doesn't mean if you haven't properly attached a correctly permissioned role to the node that it will work but the same is said for any credentials. Just because you supply keys doesn't mean they have the ability to launch ec2 instances, create security groups, etc.

Reply:
@tvpartytonight ec2.md updated